tests/snappy_pipeline/workflows/test_workflows_germline_snvs.py

```diff
@@ -191,8 +191,8 @@ def test_bcftools_step_part_get_args_pileup(germline_snvs_workflow):
             "--min-MQ 35",
             "--no-BAQ",
             "--seed 1234567",
-            "--skip-any-set 'UNMAP,SECONDARY,QCFAIL,DUP,SUPPLEMENTARY'",
-            "--skip-any-unset 'PROPER_PAIR,PAIRED'",
+            "--skip-any-set 3852",
+            "--skip-any-unset 3",
```

Comment:
Leave the flag strings as comments so it is easier to understand?
Or import the BamFlags code to convert the list of strings into the integer flag and use that?

Reply:
Yes, I'll do it